### DISCUSSION TITLE
nparray as pipeline input.

### DISCUSSION THREAD

**Discussion by sense2k8** [pyannote/pyannote-audio#871] [Q&A]
Hi,
how to use ndarray from soundfile to pipeline. Example says Pipeline takes only audio file. 

from pyannote.audio import Pipeline
pipeline = Pipeline.from_pretrained("pyannote/overlapped-speech-detection")
output = pipeline("audio.wav") <-------------------- Want to replace this with below code snippet

import soundfile as sf
waveform, sr = sf.read('sample.wav')
output = pipeline(waveform)

**Comment by hbredin** [comment]
See [this tutorial](https://github.com/pyannote/pyannote-audio/blob/develop/tutorials/applying_a_pipeline.ipynb), section "Processing a file from memory".
